### PR TITLE
Update italian translation

### PR DESCRIPTION
**Pull request by mmardegan**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -16,7 +16,7 @@ Pascal Obry
 Ger Siemerink
 EdgarLux
 Timur I. Davletshin
-Michel Leblond
```
Comment (TurboGit) [inline AUTHORS]: Unwanted change!

File: doc/usermanual/po/it.po [hunk 2]
```diff
@@ -4,9 +4,17 @@
 # Maurizio Paglia <mpaglia0@gmail.com>, 2016, 2017, 2018, 2019.
 msgid ""
 msgstr ""
-"Project-Id-Version: darktable 2.4\n"
+"Project-Id-Version: darktable 2.6\n"
 "POT-Creation-Date: 2019-03-14 20:49+0100\n"
+<<<<<<< HEAD
```
Comment (TurboGit) [inline doc/usermanual/po/it.po]: non resolved conflicts committed !

File: AUTHORS (hunk 1 above)
Comment (mmardegan) [reply AUTHORS]: It's not a my commit: i suppose it's a wrong deletion.

File: doc/usermanual/po/it.po (hunk 2 above)
Comment (mmardegan) [reply doc/usermanual/po/it.po]: I did an hard work to "rebase" the repository. This is an error that yesterday I tried to solve uploading the final version. Sorry.

**Comment by TurboGit** [comment]
Sorry I can't merge ! Now there is many duplicate entries in po/it.po ! I'm closing this. If you still have an it.po working on your side just send it to me. Thanks.

**Pull request closed without merging**